tools/src/main/scala/caliban/tools/ClientWriter.scala

```diff
@@ -376,18 +376,15 @@ object ClientWriter {
 
   def writeScalar(typedef: ScalarTypeDefinition, mappingClashedTypeNames: Map[String, String]): String =
     if (typedef.name == "Json") "type Json = io.circe.Json"
-    else
-      s"""type ${safeTypeName(typedef.name, mappingClashedTypeNames)} = String
+    else s"""type ${safeTypeName(typedef.name, mappingClashedTypeNames)} = String
         """
 
   def safeUnapplyName(name: String): String =
-    if (reservedKeywords.contains(name) || name.endsWith("_")) s"$name$$"
+    if (reservedKeywords.contains(name)) s"${name}_"
```

Comment:
I think there's a merge issue here, those changes should not be reverted and probably explain the CI error.

Reply:
Thanks for spotting that.  I will try to update that and let you know how it goes.

Reply:
Looks like the same issue with:
```
[info] [error] /tmp/sbt_f9ee2458/src/main/scala/Client.scala:82:7: overriding method wait in class Object of type ()Unit;
[info] [error]  value wait cannot override final member
[info] [error]       wait: String
[info] [error]       ^
```

I will look over the merge conflicts that I attempted to resolve and see if I screwed other spots up too.